### DISCUSSION TITLE
API and custom domain verification error

### DISCUSSION THREAD

**Discussion by gurustiven** [vercel/community#1374] [Help]
Hi there!

We have two several problems with vercel and don’t understand why are happening.

1. We use the API for launch new projects and we connect a GitLab repo (With Turborepo), but the problem is that after approximately 48hrs if we don’t  enter in the vercel.com UI admin panel, the next automatic project launched through the API will fail with a message similar to: “Repo not found in your GitLab linked account”.

2. The second big problem is also related to the first, when we attach a custom domain to any of our projects through the API, the verification process of the domain don't happen until we visit the vercel.com admin panel UI in domains section.

All our business is to automatically launch new websites (projects) and through our dashboard link with custom domain, nobody has access to Vercel.com UI.

Have you some idea about this two problems?

Thanks a lot.

**Comment by amyegan** [comment]
Can you share a deployment link or deployment id so we can look into the logs? Any more info you can share about the API endpoints used, steps you follow for deployment, and specific error messages would also be helpful 🙏
